kne/integration.textproto

```diff
@@ -8,7 +8,7 @@ nodes: {
     value: "ATE"
   }
   config: {
-    image: "magna:latest"
+    image: "magna:scale"
```

Comment:
is there a reason to have scale separate from latest? now that these fixes will be going on, why not just keep `latest` as the head. We can set `tested` or something similar if we want to maintain a separate "most-recently-confirmed-working" container?

Reply:
Agree entirely. This was simply to make sure that I didn't overwrite the `:latest` for other tests in my experiments. With this becoming `HEAD`, this can become `:latest` again.